codalab_service.py

```diff
@@ -10,9 +10,21 @@
 import errno
 import os
 import subprocess
-import sys
 
-SERVICES = ['mysql', 'nginx', 'frontend', 'rest-server', 'bundle-manager', 'worker']
+DEFAULT_SERVICES = [
+    'mysql',
+    'nginx',
+    'frontend',
+    'rest-server',
+    'bundle-manager',
+    'worker',
+    'init',
+    'update',
```

Comment:
You should have `DEFAULT_SERVICES` for the first 8 elements, and then `ALL_SERVICES = DEFAULT_SERVICES + ['test', 'monitor']`.

You need to merge master in (`update` is gone now).